CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### Bug Fixes
 
+-   Update the CI credentials. (https://github.com/pulumi/pulumi-kubernetes/pull/1097)
```

Comment:
I don't think we need a changelog entry for this. These are just user-facing changes.